unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BotsController/BotsController.cs

```diff
@@ -193,9 +197,9 @@ void InstantiateBot(Vector3 position)
             {
                 id = entityId,
                 name = entityId,
-                hairColor = Color.white,
-                eyeColor = Color.white,
-                skinColor = Color.white,
+                hairColor = Random.ColorHSV(0, 1, 0, 1, 0.25f, 0.9f),
+                eyeColor = Random.ColorHSV(0, 1, 0, 1, 0f, 0.2f),
+                skinColor = Random.ColorHSV(0, 1, 0.3f, 1, 0.4f, 0.9f),
```

Comment:
I had to test out how the rendering worked with random skin colors, so the DBZ-like skin coloring is back for the bots tool.